css-canvas-medical/styles.css

```diff
@@ -0,0 +1,278 @@
+.container {
+  width: 75%;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.column-half {
+  display: flex;
+  flex-basis: 50%;
+}
+
+nav {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  text-align: center;
+  font-size: 14pt;
+  width: 100%;
+  color: rgb(163, 163, 163);
+}
+
+.column-tenth {
+  display: flex;
+  flex-basis: 20%;
+  justify-content: center;
+  width: fit-content;
+}
+
+#nav {
+  justify-content: space-evenly;
+}
```

Comment:
Instead of having an `id` selector, why not make a class named `justify-evenly` that you can reuse?